Index: C:/Users/My PC/WebstormProjects/topcoder/.idea/topcoder.iml
===================================================================
--- C:/Users/My PC/WebstormProjects/topcoder/.idea/topcoder.iml	(date 1603418111709)
+++ C:/Users/My PC/WebstormProjects/topcoder/.idea/topcoder.iml	(date 1603418111709)
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
